src/efi.rs

```diff
@@ -484,7 +488,7 @@ pub(crate) fn set_efi_current(device: &str, espdir: &openat::Dir, vendordir: &st
     if espdir.exists(&shim)? {
         anyhow::bail!("Failed to find {SHIM}");
     }
-    let loader = format!("\\EFI\\{}\\shimx64.efi", vendordir);
+    let loader = format!("\\EFI\\{}\\{SHIM}", vendordir);
```

Comment:
@cgwalters update this to sync with `SHIM`, or might have issue on aarch64 as it does not have `shimx64.efi` when called by `--update-firmware`.

Reply:
Yes, this looks right.  I suspect the only reason it may have appeared to work on aarch64 is because we ended up using the fallback path.